libutils/fsattrs.h

```diff
@@ -0,0 +1,53 @@
+/*
+  Copyright 2025 Northern.tech AS
+
+  This file is part of CFEngine 3 - written and maintained by Northern.tech AS.
+
+  This program is free software; you can redistribute it and/or modify it
+  under the terms of the GNU General Public License as published by the
+  Free Software Foundation; version 3.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program; if not, write to the Free Software
+  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA
+
+  To the extent this program is licensed as part of the Enterprise
+  versions of CFEngine, the applicable Commercial Open Source License
+  (COSL) may apply to this file if you as a licensee so wish it. See
+  included file COSL.txt.
+*/
+
+#ifndef CFENGINE_FSATTR_H
+#define CFENGINE_FSATTR_H
+
+#include <stdbool.h>
+
+typedef enum {
+    FS_ATTRS_SUCCESS = 0,
+    FS_ATTRS_FAILURE,
+    FS_ATTRS_NOENTRY,
+    FS_ATTRS_NOTSUPP,
+} FSAttrsResult;
```

Comment:
Where are these names taken from? If you are following some naming convention, and that's why they are cryptic, I'd add a comment with a link. Otherwise, I'd rename them to more straight forward, non-abbreviated names, like `FS_ATTRS_DOES_NOT_EXIST` and `FS_ATTRS_NOT_SUPPORTED`.

Reply:
The names are used in `errno` constants. Also it's sexy when they are all the same length :wink: But I see you point and will change it to more human readable values.